src/corebluetooth.cs

```diff
@@ -43,6 +45,59 @@ interface CBCentralInitOptions {
 	interface CBManager {
 		[Export ("state", ArgumentSemantic.Assign)]
 		CBManagerState State { get; }
+
+		[iOS (13,0), TV (13,0), Watch (6,0), Mac (10, 15)]
+		[Export ("authorization", ArgumentSemantic.Assign)]
+		CBManagerAuthorization Authorization { get; }
+	}
+
+	[iOS (13,0), TV (13,0), Watch (6,0)]
```

Comment:
missing `[NoMac]` ? unless it's present (then missing `Mac (10,15, onlyOn64: true)]` or you're inside `!MONOMAC` (then better add `[NoMac]` anyway to ease review and help the IDE)